frontend/prisma/schema.prisma

```diff
@@ -15,16 +15,46 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model User {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  email     String?  @unique
-  image     String?
-  role      Role     @default(USER)
+model Clubs {
```

Comment:
can you rename all the models such that they are not plural, so Clubs -> Club, Events -> Event, makes things less confusing